types/uid-generator/uid-generator-tests.ts

```diff
@@ -1,14 +1,17 @@
-import { UIDGenerator } from 'uid-generator';
+import { default as UIDGenerator } from 'uid-generator';
 
-new UIDGenerator('abc'); // $ExpectType UIDGenerator
-const generator = new UIDGenerator(128, 'abc'); // $ExpectType UIDGenerator
+new UIDGenerator('abc'); // $ExpectType UIDGeneratorInstance
+const generator = new UIDGenerator(128, 'abc'); // $ExpectType UIDGeneratorInstance
 
 generator.generateSync(); // $ExpectType string
-generator.generate(uid => {
+generator.generate((err, uid?) => {
```

Comment:
Don't want to be picky here but you might have forgotten to remove a `?` from your previous commits

Reply:
Thanks for the reminder. Now it removed.